frontend/app/src/store/__test__/main.spec.ts

```diff
@@ -10,14 +10,22 @@ import mockBounties, { expectedBountyResponses } from '../../bounties/__mock__/m
 
 let fetchStub: sinon.SinonStub;
 
+const origFetch = global.fetch;
+
 beforeAll(() => {
   fetchStub = sinon.stub(global, 'fetch');
+  fetchStub.returns(Promise.resolve({ status: 200, json: () => Promise.resolve({}) })); // Mock a default behavior
 });
 
 afterAll(() => {
-  jest.clearAllMocks();
+  global.fetch = origFetch;
+
+  sinon.restore();
 });
 
+const getOrganizationUsersEndpoint = (orgUUID: string): string =>
+  `${TribesURL}/organizations/users/${orgUUID}`;
+

```

Comment:
I dont see the point of defining this for a single use, we can just use the string directly in the test